chef/cookbooks/nova_dashboard/templates/default/local_settings.py.erb

```diff
@@ -290,16 +290,16 @@ TIME_ZONE = "<%= @timezone %>"
 # target installation.
 
 # Path to directory containing policy.json files
-#POLICY_FILES_PATH = os.path.join(ROOT_PATH, "conf")
+POLICY_FILES_PATH = os.path.join(ROOT_PATH, '<%= @policy_file_path =>')
```

Comment:
Sorry, I guess my earlier comment was unclear. I would expect people to just upload the policies in, say, `/etc/horizon/$customer`, so the value for `@policy_file_path` could be an absolute path.

I would probably code it this way:
```
<% if @policy_file_path.empty? -%>
#POLICY_FILES_PATH = os.path.join(ROOT_PATH, "conf")
<% else -%>
POLICY_FILES_PATH = "<%= @policy_file_path =>"
<% end -%>
```

This would mean that the default value for the attribute should be empty, of course.